src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item:last-child a {
+    margin: 0;
+  }
+
+  &__link {
+    display: block;
+    line-height: 60px;
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $main-text-color;
+    margin-right: 20px;
+    transition: color 0.3s;
+
+    &:hover {
+      color: $blue-brand-color;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $blue-brand-color;
+
+  &::after {
+    content: '';
+    position: absolute;
+    display: block;
+    height: 4px;
+    width: 100%;
+    background-color: $blue-brand-color;
+    border-radius: 8px;
+    bottom: 0;
+  }
+}
```

Comment:
The `.is-active` class seems to be a modifier for the `nav__link`, but it's defined as a separate block. According to BEM methodology and the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), modifiers should be named using the block or element name followed by double dashes. Consider renaming `.is-active` to `.nav__link--is-active` to make it clear that it's a modifier of `.nav__link`.